### DISCUSSION TITLE
蓝奏云优享版

### DISCUSSION THREAD

**Discussion by wxz-12138** [AlistGo/alist#6169] [General]
蓝奏云优享版 提示“failed get objs: failed to list objs: 超时链接 -1，请重新输入”是不能用了吗？

**Comment by Greyisttot** [comment]
https://github.com/alist-org/alist/issues/6171

**Comment by ee44** [comment]
确实是这样了，  
 failed get objs: failed to list objs: -1: 超时请求

大佬解决一下吧

**Comment by huolanyun** [comment]
小飞机盘也也一样，两个登陆协议一样

**Comment by im-dashan** [comment]
我也是一样，希望尽快解决
